geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/RegionFunctionArgs.java

```diff
@@ -38,165 +37,181 @@
 public class RegionFunctionArgs implements Serializable {
   private static final long serialVersionUID = -5158224572470173267L;
```

Comment:
I think we need to change the serialVersionUID for this change.  Also, I'm a bit concerned about the effect that will have on backwards compatibility.  I know that we require the gfsh version to match the version of the locator /jmxManager it is connecting to, but I think a user can have something like Gfsh and Locator versions 1.3 but servers still on version 1.2, and I think this would cause backwards compatibility (serialization) problems in that case.  I wonder if we can find any other examples of changes to functions that back gfsh commands to see how it was dealt with?